<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaultTopologyWidget</class>
 <widget class="QWidget" name="FaultTopologyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fault Topology Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="faultLayerLabel">
       <property name="text">
        <string>Fault Layer:</string>
       </property>
      </widget>
     </item>
    <item row="0" column="1">
     <widget class="QgsMapLayerComboBox" name="faultLayerComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="faultIdFieldLabel">
      <property name="text">
       <string>Fault ID Field:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QgsFieldComboBox" name="faultIdFieldComboBox"/>
    </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Calculate Topology</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
   <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
